ftl/core/importing.ftl

```diff
@@ -116,6 +116,54 @@ importing-cards-added =
        *[other] { $count } cards added.
     }
 importing-file-empty = The file you selected is empty.
+importing-notes-added =
+    { $count ->
+        [one] { $count } new note was successfully imported into your collection.
+       *[other] { $count } new notes were successfully imported into your collection.
+    }
+importing-notes-updated =
+    { $count ->
+        [one] { $count } was updated to the latest version.
+       *[other] { $count } were updated to the latest version.
+    }
+importing-existing-notes-skipped =
+    { $count ->
+        [one] { $count } was not imported because it already exists identically in your collection.
+       *[other] { $count } were not imported because they already exist identically in your collection.
+    }
+importing-conflicting-notes-skipped =
+    { $count ->
+        [one] { $count } was not imported because its note type has changed.
+       *[other] { $count } were not imported because their note type has changed.
+    }
+importing-import-log = Import Log
+importing-import-log-for-file = Import Log for { $val }
```

Comment:
We probably don't need it if we pass the path in MnemosyneImporter to .import_json_string()

Reply:
How important is it to show the filename in the title - is it likely the user won't know which file they imported?

Reply:
Seems useful to me because sometimes users download quite a few decks all at once and import them in short succession afterwards (or even select them all and hit Enter). In that case it would be useful to know which file the log refers to.

Another option might be:
`Import Log - [filename]`